ulib/Makefile.verify

```diff
@@ -34,3 +34,34 @@ LowStar.Printf.fst.checked: USE_EXTRACTED_INTERFACES=
 verify-core: $(addsuffix .checked, $(filter-out $(EXTRA) ,$(FSTAR_FILES)))
 
 verify-extra: $(addsuffix .checked, $(EXTRA))
+
+# Benchmarking rules
+#
+# we want to run FStar with just the target file as that being checked
+# ideally all the .checked files will exist and we will:
+#  - move them to the side
+#  - execute the benchmark
+#  - move the checked file back
+#
+
+# a.fst.bench will move the a.fst.checked binary and then move it back
+%.fst.bench: %.fst
+	-mv -f $*.fst.checked $*.fst.checked.bench_bkp
+ifdef BENCHMARK_FSTAR
```

Comment:
depending on whether you're ok with .bench files that don't have the exact right name, you could also use BENCHMARK_PRE here and get rid of the ifdef

Reply:
I think I've got a solution for this in the BENCHMARK_PRE rule, getting rid of the ifdef is more important.